src/NuGet.Core/NuGet.Protocol/NuGet.Protocol.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>$(TargetFrameworksLibraryForSigning)</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworksLibraryForSigning);$(NETCoreTargetFramework)</TargetFrameworks>
```

Comment:
I am curious to learn more about why we need the additional framework.

Reply:
Looks like the new tfm is used while checking if a file is executable on non-Windows platform. https://github.com/NuGet/NuGet.Client/pull/5990/files#diff-cf421e9b689e2c0e46c47e27365571384c9fd9c7eae6522a36c26f5bd40312a4R193